index.d.ts

```diff
@@ -1,10 +1,16 @@
 declare module "@iarna/toml" {
+  type AnyJson = boolean | number | string | null | JsonArray | JsonMap
+  interface JsonArray extends Array<AnyJson> {}
+  interface JsonMap {
+    [key: string]: AnyJson;
+  }
+  
   interface FuncParse {
-    (toml: string): object
+    (toml: string): JsonMap
```

Comment:
maybe

```diff
-(toml: string): JsonMap
+<T = JsonMap>(toml: string): T
```

that way you could specify the definite type of whatever gets parsed if you know what it will be. Same goes for `async` and `stream`. What do you think?

Reply:
Sorry for a newbee to TOML. Is it possible for `parse` to return a non-Object (key-value pair, i.e. JsonMap) value?
I've tried to play with the package in the REPL and seems that TOML should always contain key-value pairs.

Reply:
Your understanding is correct, it can only ever return key value pairs.

Reply:
From the emitting side, `TOML.stringify.value` can generate just the value part (which can be a any TOML type), but at the moment there's no parse equivalent. If there is, it will be on a different API as it'll be working on fragments of TOML instead of complete documents.